gnovm/tests/files/type37.gno

```diff
@@ -0,0 +1,20 @@
+package main
+
+import "fmt"
+
+type IntArray []int
+type Arr *IntArray
```

Comment:
Can you add a test where `Arr` is an argument to a function and make sure all the `*[]T` semantics work?

Reply:
this is added. type38.gno